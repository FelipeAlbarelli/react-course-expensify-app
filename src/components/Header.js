import React from 'react';
import {NavLink} from 'react-router-dom'

const Header = () => (
    <header>
        <h1>Expensify</h1>
            <NavLink to='/' exact activeClassName='is-active'>Dashboard</NavLink>
            <NavLink to='/create' activeClassName='is-active'>Create</NavLink>
            <NavLink to='/help' activeClassName='is-active'>Help</NavLink>
            <NavLink to='/24234234' activeClassName='is-active'>404</NavLink>
    </header>
)

export default Header