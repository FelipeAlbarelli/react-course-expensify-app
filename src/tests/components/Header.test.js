import React from 'react';
import Header  from '../../components/Header';
// import ReactShallowRenderer from 'react-test-renderer/shallow'
import {shallow} from 'enzyme'

// test('should render Header correctly' , () =>{
//     const renderer = new ReactShallowRenderer();
//     renderer.render(<Header />)
//     console.log(renderer.getRenderOutput())
// })

test('should render Header correctly' , () => {
    const wrapper = shallow(<Header />);
    expect((wrapper)).toMatchSnapshot();
})





