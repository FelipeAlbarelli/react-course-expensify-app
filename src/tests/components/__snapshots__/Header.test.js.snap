// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Create
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
    Help
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/24234234"
  >
    404
  </NavLink>
</header>
`;
